deltachat-rpc-client/src/deltachat_rpc_client/pytestplugin.py

```diff
@@ -197,3 +214,134 @@ def indent(self, msg: str) -> None:
             print("  " + msg)
 
     return Printer()
+
+
+#
+# support for testing against different deltachat-rpc-server/clients
+# installed into a temporary virtualenv and connected via 'execnet' channels
+#
+
+
+def find_path(venv, name):
+    is_windows = platform.system() == "Windows"
```

Comment:
So it already works on Windows, the PR description is stale apparently?

Reply:
yes, it was stale. fixed the PR description.